htlcswitch/link.go

```diff
@@ -241,6 +241,11 @@ type ChannelLinkConfig struct {
 	// in testing, it is here to ensure the sphinx replay detection on the
 	// receiving node is persistent.
 	UnsafeReplay bool
+
+	// UpdateFeeInterval is the interval at which the update fee timer will
+	// fire at in order to determine if the link should update its
+	// commitment fee.
+	UpdateFeeInterval func() time.Duration
```

Comment:
Why does this need to be a method? It can simply be an attribute.

Reply:
Actually, alternatively, it can be a min/max. Don't have a strong opinion in either direction.

Reply:
Fixed.